storage/balance_storage.go

```diff
@@ -521,3 +521,44 @@ func (b *BalanceStorage) GetAllAccountCurrency(
 
 	return accounts, nil
 }
+
+// SetBalanceImported sets the balances of a set of addresses by
+// getting their balances from the tip block, and populating the database.
+// This is used when importing prefunded addresses.
+func (b *BalanceStorage) SetBalanceImported(
+	ctx context.Context,
+	helper BalanceStorageHelper,
+	accountBalances []*utils.AccountBalance,
```

Comment:
Why create a new type for this? Why not just use `*types.AccountBalanceResponse`?